keycloak/lib/all-stacks.ts

```diff
@@ -0,0 +1,126 @@
+    const utilsStack = new KeycloakUtils(this, 'KeyCloakUtils', {
+      hostedZone: AllStacks.HOSTED_ZONE,
+      internalHostedZone: AllStacks.INTERNAL_HOSTED_ZONE,
+    });
+
+    // Create RDS database
+    const rdsDBStack = new RdsStack(this, 'KeycloakRDS', {
+      vpc: vpcStack.vpc,
+      rdsDbSecurityGroup: vpcStack.rdsDbSecurityGroup,
+      rdsAdminPassword: utilsStack.keycloakDbPassword,
+    });
+    rdsDBStack.node.addDependency(vpcStack, utilsStack);
+
+    // Deploy and install Public KeyCloak on EC2
+    const keycloakStack = new KeycloakStack(this, 'public', {
+      vpc: vpcStack.vpc,
+      keycloakSecurityGroup: vpcStack.keyCloaksecurityGroup,
+      certificateArn: utilsStack.certificateArn,
+      hostedZone: utilsStack.zone,
+      initConfig: AllStacks.getCfnInitConfigForPublicKeycloak(this.region, {
+        rdsInstanceEndpoint: rdsDBStack.rdsInstanceEndpoint,
+        keycloakDBpasswordSecretArn: utilsStack.keycloakDbPassword.secretFullArn,
+        keycloakCertPemSecretArn: utilsStack.keycloakCertPemSecretArn,
+        keycloakCertKeySecretArn: utilsStack.keycloakCertKeySecretArn,
+      }),
+    });
+
+    keycloakStack.node.addDependency(vpcStack, rdsDBStack, utilsStack);
+
+    // Deploy and install Internal KeyCloak on EC2
+    const keycloakInternalStack = new KeycloakStack(this, 'internal', {
+      vpc: vpcStack.vpc,
+      keycloakSecurityGroup: vpcStack.keycloakInternalSecurityGroup,
+      certificateArn: utilsStack.internalCertificateArn,
+      hostedZone: utilsStack.internalZone,
+      initConfig: AllStacks.getCfnInitConfigForInternalKeycloak(this.region, {
+        rdsInstanceEndpoint: rdsDBStack.rdsInstanceEndpoint,
+        keycloakDBpasswordSecretArn: utilsStack.keycloakDbPassword.secretFullArn,
+        keycloakAdminUserSecretArn: utilsStack.keycloakAdminUserSecretArn,
+        keycloakAdminPasswordSecretArn: utilsStack.keycloakAdminPasswordSecretArn,
+        keycloakCertPemSecretArn: utilsStack.keycloakCertPemSecretArn,
+        keycloakCertKeySecretArn: utilsStack.keycloakCertKeySecretArn,
+      }),
+    });
+
+    keycloakInternalStack.node.addDependency(vpcStack, rdsDBStack, utilsStack);
+
+    // Create WAF stack
+    const wafStack = new KeycloakWAF(this, 'KeycloakWAFstack', {
+      loadBalancerArn: keycloakStack.loadBalancerArn,
+      internalLoadBalancerArn: keycloakInternalStack.loadBalancerArn,
+    });
+
+    wafStack.node.addDependency(keycloakStack);
+  }
+
+  private static getCfnInitConfigForPublicKeycloak(region: string, props: InitProps): InitElement[] {
+    return [
+      InitPackage.yum('docker'),
+      InitCommand.shellCommand('sudo curl -L https://github.com/docker/compose/releases/download/v2.9.0/docker-compose-$(uname -s)-$(uname -m) '
+        + '-o /usr/bin/docker-compose && sudo chmod +x /usr/bin/docker-compose'),
+      InitFile.fromFileInline('/docker-compose.yml', join(__dirname, '../../resources/docker-compose.yml')),
```

Comment:
path is wrong. should be `../resources/docker-compose.yml`.

Reply:
Fixed!